src/DocumentFormat.OpenXml/Validation/ValidationContext.cs

```diff
@@ -47,7 +49,15 @@ public ValidationContext(ValidationSettings settings, ValidationCache cache)
 
         public bool Valid => Errors.Count == 0;
 
-        public bool IsCancelled => MaxNumberOfErrors > 0 && Errors.Count >= MaxNumberOfErrors;
+        public bool IsCancelled
+        {
+            get
+            {
+                _token.ThrowIfCancellationRequested();
```

Comment:
I do not recommend throwing an exception in the get method. From the name of this property, we may not expect any exception selling.

Reply:
Hmm good point as to not throwing there. I'll change it to a method